pkg/workloads/cortex/lib/storage/local.py

```diff
@@ -141,3 +141,6 @@ def download_and_unzip(self, key, local_dir):
         local_zip = os.path.join(local_dir, "zip.zip")
         self.download_file(key, local_zip)
         util.extract_zip(local_zip, delete_zip_file=True)
+
+    def list_objects(self, path):
+        return os.listdir(path)
```

Comment:
This isn't exactly the same behavior as the s3 one, since it should return file paths of all files at or below the directory. I think `os.walk()` can help.